script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -676,63 +680,63 @@ That reminds me.
 [sha:9ef905c968a33ad8ae3f37c3e9f81b29c5d8d23b]{
 -- Page 613, Offset 35031.
 -- いま、あの透ける壁の向こうに、わたしに気付いて助けを呼ぶ、お母さんの顔が、あったような。
-Just now, on the other side of the transparent wall, I felt like I saw my mother, crying for help after noticing me.
+I think... I just saw my mother on the other side of the transparent wall, begging for help the moment she noticed me.
 }
 [sha:8cda461afb782769a08529a3f319b5b75a0c27ed]{
 -- Page 614, Offset 35032.
 -- 逃げる途中、色々なものを見た。
-There were all sorts of things we saw as we ran.
+// are these recounting segments supposed to be in past or present in en? idrk the precedent
+We saw all sorts of things as we ran.
 }
 [sha:70548eef14c2e165a80dd6ed56c285dc15d609db]{
 -- Page 615, Offset 35033.
 -- たとえば一面の展覧会。
-For example, on one side, there was an exhibition.
+For example, on one side, there's an exhibition.
 }
 [sha:c1c1f78569bfa19ad8ebb9114de6dd4e0e5e071f]{
 -- Page 615, Offset 35034.
 -- 広場に続く道路には幾つものオブジェが立っていた。よく見るとそれは人間の大きさぐらいのサボテンで、もっとよく見ると人間そのものだった。
-// lint-off:LintPageOverflow
-On the road to the square, there were many pieces of art standing up. When you look closely, you could see they were cacti around the size of a human, and if you look even closer, you could see that they were humans themselves.
+On the road to the square, countless pieces of art have been erected. Looking closely, I realize that they're human-sized cacti. And then, looking even closer, I realize that they aren't actually cacti, but people.
 }
 [sha:faa7c1c88469e4eb03a26de4ef0969f385b8159d]{
 -- Page 615, Offset 35035.
 -- 彼らの表面は<茨|いばら>のようなものでグルグル巻きで、サボテンに見えたのも仕方がない。
-They were turned and twisted like thorns on their surface, so it was only natural that I mistook them for cacti.
+Their bodies are all twisted up like thorns, so it's only natural that I mistook them for cacti.
 }
 [sha:ce85f4ef241a88c330883e0e55170c6e4a1352dc]{
 -- Page 615, Offset 35036.
 -- そのサボテンには真っ赤な花が咲いていた。人間を苗床にして咲く吸血花。血液で咲く<深紅の薔薇|ルージュメイアン>。
-A deep red flower bloomed from these cacti. A blood-sucking flower that used humans as a nursery. A crimson rose that blooms through their blood, Rouge Mailland.
+A deep red flower bloomed from these cacti. A vampiric flower that uses humans as a seedbed. A flower that blossoms by sucking human blood, Rouge Mailland.
```

Comment:
again, this random french is just nonsense if you don't speak french. This one definitely should be ruby